app/src/components/notifications/PageNotifications.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <TmPage
     data-title="My alerts"
-    :loading="$apollo.queries.notifications.loading"
+    :loading="$apollo.queries.notifications.loading && !firstLoaded"
```

Comment:
not your fault ana, but i think this is still confusing logic. 

we have `loading` and `loading-more` props + `firstLoaded` `dataLoaded` `moreAvailable`

not necessarily in this PR but maybe we can simplify to:

`firstLoad`
`loadMore`

and use computed properties instead of `$apollo.queries.notifications.loading && !dataLoaded && moreAvailable` in the template?

Reply:
true, we could use `firstLoaded` for loading and maybe `dataLoaded` for `loading-more`. Let me try that